### PR TITLE
Update inputs and pool card

### PR DESCRIPTION
**Pull request by totop716**
- Create custom component to input address and use this for recipient input
- Replace transaction deadline input with numerical input.
- Fix `View Analytics` button is redirecting to wrong link when lp farm does not exist on Pool card
- Show APY on Pool card when dual farm exists for the pair.

### CODE REVIEW

File: src/components/Swap/Swap.tsx [hunk 1]
```diff
@@ -628,21 +612,18 @@ const Swap: React.FC<{
...
           {recipient !== null && (
-            <Box className='content'>
-              <Typography>Recipient</Typography>
-              <input
-                value={recipient}
-                placeholder='Wallet Address or ENS name'
-                onChange={(evt) => onChangeRecipient(evt.target.value)}
-              />
-            </Box>
+            <AddressInput
```
Comment (FriskyHamTitz) [inline src/components/Swap/Swap.tsx:621]: We should check if the address is ENS, or implement a checksum to validate the address.  

https://ethereum.stackexchange.com/questions/1374/how-can-i-check-if-an-ethereum-address-is-valid

Comment (FriskyHamTitz) [reply src/components/Swap/Swap.tsx:621]: that logic is in solidity but should be applicable to javascript

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]

**Comment by FriskyHamTitz** [comment]
Please add a checksum for addresses

File: src/components/Swap/Swap.tsx (hunk 1 above)
Comment (totop716) [reply src/components/Swap/Swap.tsx:621]: We are checking if the address is checksummed in `useENS` hook.

**Review by FriskyHamTitz — APPROVED** [review]
Just make sure you test first

**Pull request merged**